tools/__tasks__/validate/javascript.js

```diff
@@ -22,15 +22,14 @@ const dirs = p =>
 module.exports = {
     description: 'Lint JS',
     task: [
-        ...dirs('static/src/javascripts')
-            .map(dir => ({
-                description: `App ${chalk.dim(dir)}`,
-                task: `eslint static/src/javascripts/${dir} ${config}`,
-                onError: error,
-            })),
+        {
+            description: 'Static',
+            task: `eslint static/src/javascripts --ext=ts,tsx,js --no-error-on-unmatched-pattern ${config}`,
```

Comment:
I think this is the right reason?

```suggestion
// --no-error-on-unmatched-pattern is required for linting before converting to any files to TS, it can be removed after conversion of any files
            task: `eslint static/src/javascripts --ext=ts,tsx,js --no-error-on-unmatched-pattern ${config}`,
```

Reply:
maybe we should have one ts file in the feature branch so we don't need this? just incase it ends up not matching our to error and nothing is listed